integration-tests/tests/src/tests/observable.ts

```diff
@@ -39,61 +39,127 @@ function expectObservableMethods(observable: Observable) {
   expect(observable.removeAllListeners).to.be.a("function");
 }
 
+type Action = () => void;
+type ChangeAndActions<ChangeSet extends object> = { expectedChange: ChangeSet; actions: Action[] };
```

Comment:
It doesn't matter too much what it is, as long as it's not a `function` since that's what I use to distinguish actions from changesets as `typeof fn === "function"` for functions and `typeof changeset === "obejct"` for the changeset options 👍